administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -33,7 +62,118 @@ function triggerSave() {
 			$('#left-panel').toggle();
 			$('#right-panel').toggleClass('full-width');
 		});
+
```

Comment:
To preload the image in the loading layer, add, at the beggining of the document ready event

```js
// Create and append the loading layer.
Joomla.loadingLayer("load");
```